bot.go

```diff
@@ -89,8 +97,8 @@ func updateMap(msgID, replyID string) {
 	}
 }
 
-func setFormatMode(session *discordgo.Session, message *discordgo.Message, p *CommandParams) {
-	message.Content = tagToCmd(message.Content, session.State.User.ID)
+func setFormatMode(message *discordgo.Message, p *CommandParams) {
+	message.Content = tagToCmd(message.Content, selfID)
```

Comment:
global variables are evil... but quite handy in a bot

I should probably move everything into BotState struct though but then would need to lambda-ify all the callbacks